upgradeWindowsVHD causes non windows upgrade commands to fail

**Describe the bug**

`upgradeWindowsVHD` was added as part of this PR - https://github.com/Azure/aks-engine/pull/3774. This doesn't check if the windows profile exists before accessing it which causes upgrade commands to panic and fail. 

```bash
root@b3a33ed8751b:/aks-engine# ./bin/aks-engine upgrade --subscription-id <sub id> --api-model _output/akse0909/apimodel.json --resource-group akse0909 --upgrade-version 1.18.6 --location southcentralus --auth-method cli
WARN[0000] No "location" value was specified, AKS Engine will generate an ARM template configuration valid for regions in public cloud only 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x78 pc=0x1609174]

goroutine 1 [running]:
github.com/Azure/aks-engine/cmd.(*upgradeCmd).loadCluster(0xc0000a60f0, 0x0, 0x0)
        /aks-engine/cmd/upgrade.go:177 +0x7b4
github.com/Azure/aks-engine/cmd.(*upgradeCmd).run(0xc0000a60f0, 0xc000149400, 0xc00021a0c0, 0x0, 0xc, 0x0, 0x0)
        /aks-engine/cmd/upgrade.go:287 +0x74
github.com/spf13/cobra.(*Command).execute(0xc000149400, 0xc00021a000, 0xc, 0xc, 0xc000149400, 0xc00021a000)
        /aks-engine/vendor/github.com/spf13/cobra/command.go:762 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc0004f0500, 0x1bec9a0, 0xc000128008, 0xc00010e058)
        /aks-engine/vendor/github.com/spf13/cobra/command.go:852 +0x2ea
github.com/spf13/cobra.(*Command).Execute(...)
        /aks-engine/vendor/github.com/spf13/cobra/command.go:800
main.main()
        /aks-engine/main.go:17 +0x8e
```

**Steps To Reproduce**
<!-- Please include the API model used to deploy the cluster if applicable (make sure to redact any secrets) -->

**Expected behavior**

**AKS Engine version**
master

**Kubernetes version**

**Additional context**

--- a/cmd/upgrade.go
+++ b/cmd/upgrade.go
@@ -171,7 +171,7 @@ func (uc *upgradeCmd) loadCluster() error {
 	}
 
 	// Use the Windows VHD associated with the aks-engine version if upgradeWindowsVHD is set to "true"
-	if uc.upgradeWindowsVHD {
+	if uc.upgradeWindowsVHD && uc.containerService.Properties.WindowsProfile != nil {
 		windowsProfile := uc.containerService.Properties.WindowsProfile
 		if windowsProfile.WindowsPublisher == api.AKSWindowsServer2019OSImageConfig.ImagePublisher && windowsProfile.WindowsOffer == api.AKSWindowsServer2019OSImageConfig.ImageOffer {
 			windowsProfile.ImageVersion = api.AKSWindowsServer2019OSImageConfig.ImageVersion

